### DISCUSSION TITLE
Can I add a protobuf target alone?

### DISCUSSION THREAD

**Discussion by logerrors** [xmake-io/xmake#2540] [Q&A]
In my scene, I want  to add a protobuf target alone, then parent target add add it as a dep.
![CleanShot 2022-07-06 at 17 22 40](https://user-images.githubusercontent.com/31824303/177517533-849e08e5-59f4-4a72-a939-1e2657a1a985.png)
but it seem to not work...
![CleanShot 2022-07-06 at 17 24 03](https://user-images.githubusercontent.com/31824303/177517701-92607813-13a9-4a5c-8a30-e3afe8a1bc80.png)
nothing happend
![CleanShot 2022-07-06 at 17 25 07](https://user-images.githubusercontent.com/31824303/177517923-6de2c872-a595-47bd-9d82-cce87a995bcf.png)

**Comment by waruqi** [comment]
I'm not sure it's supported, you could try using set_kind("object"). Then, add an empty .c file to the proto target
